### ISSUE TITLE
 has no method 'observe'

### ISSUE THREAD

**Issue by petermein** [athombv/athom-cli#1]
After installing athom-cli running any of the commands results in `has no method 'observe'`

Im running OSX 10.10.4 and installed athom-cli globally. 

Stacktrace:
```
/usr/local/lib/node_modules/athom-cli/node_modules/jsop/node_modules/observed/lib/observed.js:77
  Object.observe(this.subject, this.onchange);
         ^
TypeError: Object function Object() { [native code] } has no method 'observe'
    at Observable._bind (/usr/local/lib/node_modules/athom-cli/node_modules/jsop/node_modules/observed/lib/observed.js:77:10)
    at new Observable (/usr/local/lib/node_modules/athom-cli/node_modules/jsop/node_modules/observed/lib/observed.js:41:8)
    at Observable (/usr/local/lib/node_modules/athom-cli/node_modules/jsop/node_modules/observed/lib/observed.js:36:12)
    at module.exports (/usr/local/lib/node_modules/athom-cli/node_modules/jsop/index.js:13:14)
    at Object.<anonymous> (/usr/local/lib/node_modules/athom-cli/lib/settings.js:10:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by WeeJeWel** [comment]
Which nodejs version are you running?

**Comment by petermein** [comment]
I'm running `v0.10.29`

**Comment by WeeJeWel** [comment]
Please upgrade to 0.12.x

**Comment by petermein** [comment]
That fixed it. Thanks. 

Maybe a good thing to add a minimum node version to the requirement.

**Comment by WeeJeWel** [comment]
Hm, it's in the engines entry in package.json. I thought that node would've
prevented it from running, but I guess I have to add a check myself.

**Comment by petermein** [comment]
Here is a good resource about the behaviour of the engine en engineStrict variable in the package.json 
http://www.marcusoft.net/2015/03/packagejson-and-engines-and-enginestrict.html

**Comment by WeeJeWel** [comment]
Fixed in 67099cd03667f63423f3bb2fd372f79d67dc8018
